view/app/settings/domains/components/update-domain.tsx

```diff
@@ -81,11 +81,6 @@ function UpdateDomainDialog({ open, setOpen, id, data }: UpdateDomainDialogProps
 
   return (
     <Dialog open={open} onOpenChange={setOpen}>
-      {!id && (
-        <DialogTrigger asChild>
-          <Button variant="outline">{t('settings.domains.update.addButton')}</Button>
-        </DialogTrigger>
-      )}
```

Comment:
Nice catch!! 🙌🏻 

@Raj-G07: Can you remove the import for `DialogTrigger` since it is not used anywhere within the file?

LGTM.

---

Thank you so much your time and contribution towards the project! Really appreciate your efforts!! 🥇  

Look forward to more contributions!

Happy Learning!